### ISSUE TITLE
Missing Headers

### ISSUE THREAD

**Issue by foochuanyue** [twinssbc/Ionic-Calendar#52]
Hello I may be new to this so please guide me.

I tried integrating the calendar into my app, however in the header there is no options to change the view nor does it show the date.

Am I missing some part of the code?

My code looks something like this:
<ion-view view-title="Calendar" cache-view="true">
  <ion-content class="has-header">
  <calendar calendar-mode="mode" event-source="eventSource">
  </calendar>
  </ion-content>
</ion-view>

**Comment by twinssbc** [comment]
@foochuanyue Yes, the header in the demo page is not part of this calendar directive.
You can build your own layout in the header to control the calendar.
You can refer to the demo page as an example.
